secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -383,6 +418,15 @@ public void testUpdateSecret() throws IOException {
     assertThat(stdOut.toString()).contains("Updated secret");
   }
 
+  @Test
+  public void testCreateUpdateSecretLabel() throws IOException {
+    SecretName name = SecretName.parse(TEST_SECRET.getName());
+    CreateUpdateSecretLabel.createUpdateSecretLabel(
+        name.getProject(), name.getSecret(), UPDATED_LABEL_KEY, UPDATED_LABEL_VALUE);
+
+    assertThat(stdOut.toString()).contains("Updated secret");
```

Comment:
Instead of the stdOut, use the returned value from the sample to assert if the label has been updated.